packages/client/hmi-client/src/components/workflow/tera-workflow-node.vue

```diff
@@ -138,7 +145,7 @@ function showNodeDrilldown() {
 		if (pageType && assetId) {
 			openedWorkflowNodeStore.set(assetId, pageType);
 		}
-	} else alert('Node needs a valid output');
+	} else logger.error('Node needs a valid output');
```

Comment:
I'd probably make this silent `logger.error('Node needs a valid output', { silent: true }`